### DISCUSSION TITLE
Record function without using extra encoder

### DISCUSSION THREAD

**Discussion by dwi110** [DistroAV/DistroAV#1080] [Ideas]
I was wondering if it's possible to record the incoming video and audio stream of an ndi source "raw". What i mean with raw is, without using an encoder but without having ultra large files, because incoming ndi stream (especially hx) is only 5 to 20 mbit. 

My use case would be recording up to 10 ndi sources without to much of an impact on the hardware. 

Is that theoretically possible?

**Comment by Trouffman** [comment]
This is possibvle via the ISO Recorder or the NDI Studio monitor. Not something that will be implemented in the plugin.
